### PR TITLE
Changelog v51.14.0

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?

Changelog for release v51.14.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+# 51.14.0 - May 21, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Enable diff detection for folders ([PR #3243](https://github.com/forcedotcom/salesforcedx-vscode/pull/3243))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: @sbudhirajadoc whenever you get a chance to review - let me know if you have any questions! Wasn't sure if we had some more details we wanted to include for Diff Detection?

Comment (sbudhirajadoc) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: The language in the change log is not consistent so we could go either way at this point, but did you want to say "Enabled", maybe?

Comment (AnanyaJha) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: Yeah “Enabled” sounds good to me

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+- Add diff detection for all metadata types ([PR #3240](https://github.com/forcedotcom/salesforcedx-vscode/pull/3240))
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]: For consistency, how about "Enabled" here as well?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+#### salesforcedx-vscode-lwc
+
+- Add support for new LWC Analytics Dashboard feature and fix some under-the-hood bugs ([PR #3232](https://github.com/forcedotcom/salesforcedx-vscode/pull/3232))
+
+#### docs
+
+- Update docs for Diff Detection([PR #3250](https://github.com/forcedotcom/salesforcedx-vscode/pull/3250))
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]: How about "Doc updates for Diff Detection" ?

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: @sbudhirajadoc Actually it looks like we've been using present tense for the other changelog entries, so should we leave it as `Enable`?

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]:
```suggestion
- Enable diff detection for all metadata types ([PR #3240](https://github.com/forcedotcom/salesforcedx-vscode/pull/3240))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 3 above)
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]:
```suggestion
- Doc updates for Diff Detection([PR #3250](https://github.com/forcedotcom/salesforcedx-vscode/pull/3250))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -45,6 +63,7 @@
 #### salesforcedx-vscode-core
 
 - Fix issues with the following types for deploy/retrieve library ([PR #3147](https://github.com/forcedotcom/salesforcedx-vscode/pull/3147), [Issue #3114](https://github.com/forcedotcom/salesforcedx-vscode/issues/3114), [Issue #3157](https://github.com/forcedotcom/salesforcedx-vscode/issues/3157)):
+
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:66]:
```suggestion
```

**Review by jag-sfdc — APPROVED** [review]

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (sbudhirajadoc) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: I looked at older updates and we seem to have a hodge-podge of both present and past tenses. Past tense is preferred (Fixed, Updated, Resolved, Added, Improved etc). I am fine with leaving it in present tense. I will work on a template that we can use in the future logs.

Comment (AnanyaJha) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: Sounds good to me, a template would be helpful!

**Pull request merged**